event-buffer/event-buffer-core/src/main/java/uk/gov/justice/services/event/buffer/core/repository/subscription/StreamStatusJdbcRepository.java

```diff
@@ -64,27 +65,22 @@ private void initialiseDataSource() {
      */
     public void insert(final Subscription subscription) {
         try (final PreparedStatementWrapper ps = preparedStatementWrapperFactory.preparedStatementWrapperOf(dataSource, INSERT)) {
-            ps.setLong(1, subscription.getPosition());
-            ps.setObject(2, subscription.getStreamId());
-            ps.setString(3, subscription.getSource());
-            ps.executeUpdate();
+            setPreparedStatementParameters(subscription, ps);
```

Comment:
I liked this method in-lined better...easier to look at a glance what is happening

Reply:
+1